src/plugins/xerces/README.md

```diff
@@ -64,10 +64,12 @@ the mount point, then it uses the mount point's name instead.
 
 ## Dependencies
 
-- `Xerces-C++ 3.0.0` or newer (`apt-get install libxerces-c-dev`)
+- `Xerces-C++ 3.0.0` or newer (`apt-get install libxerces-c-dev` or `brew install xerces-c` on macOS)
 - CMake 3.6 or a copy of `FindXercesC.cmake` in
   `/usr/share/cmake-3.0/Modules/`
 
+To include this plugin in a homebrew installation run `brew install elektra --with-dep-plugins`
```

Comment:
This will not work, if you use the [official Homebrew formula for Elektra](https://github.com/Homebrew/homebrew-core/blob/master/Formula/elektra.rb). I also fixed how the installation works in the [most recent commits to `homebrew-elektra`](https://github.com/ElektraInitiative/homebrew-elektra/compare/afab17c..ea1d4ce). To install Elektra with the Xerces plugin, please use the commands below.

```sh
brew tap elektrainitiative/elektra
brew install elektrainitiative/elektra/elektra --with-xerces
```

Reply:
I updated the description accordingly, may I ask why there are two different taps? I'm not that familiar with homebrew and therefore a bit curios.

Reply:
> I updated the description accordingly, may I ask why there are two different taps?

The maintainers of [Homebrew’s main repository](https://github.com/Homebrew/homebrew-core) do not like formulas with (many) options that much. That is also why it took [three attempts](https://github.com/Homebrew/homebrew-core/pulls?utf8=✓&q=Elektra) before the formula for Elektra was merged. Our [own tap](https://github.com/ElektraInitiative/homebrew-elektra) allows us to support any custom modifications. This means we can support more plugins, bindings and tools that require dependencies.

Reply:
Thank you for the background info!